src/app/cluster/list/external-cluster/component.ts

```diff
@@ -219,6 +222,27 @@ export class ExternalClusterListComponent implements OnInit, OnChanges, OnDestro
     this._router.navigate(['projects', this._selectedProject.id, 'external-cluster-wizard']);
   }
 
+  onActivate(component: QuotaWidgetComponent): void {
+    component.isExternalCluster = true;
+    component.showDetailsOnHover = false;
+    this._projectService.onProjectChange
+      .pipe(startWith(this._selectedProject), takeUntil(this._unsubscribe))
+      .subscribe(({id}) => {
+        component.projectId = id;
+      });
+  }
+
+  onActivateQuotaWidgetWithoutCard(component: QuotaWidgetComponent): void {
+    component.isImportedCluster = true;
+    component.showDetailsOnHover = false;
+    component.showAsCard = false;
+    this._projectService.onProjectChange
```

Comment:
making `isExternalCluster` true by default will impact other methods, where I'll have to make it false rather than true here, that would result in more code then here, thoughts?

Reply:
yeah! i thought the same than already existing implementation you need to provide extra input as `false` to other places where it is not required.

thoughts?

```suggestion
onActivate(component: QuotaWidgetComponent, showAsCard: boolean, isExternalCluster): void {
       //// isExternalCluster -> true and false as Imported?
    }
```

Reply:
correct!

Reply:
okay leave it as separate for for code readability